rust/Cargo.toml

```diff
@@ -7,9 +7,25 @@
 # Author: Frédéric GARDES <frederic.gardes@orange.com> et al.
 # Software description: This Intelligent Transportation Systems (ITS) [MQTT](https://mqtt.org/) client based on the [JSon](https://www.json.org) [ETSI](https://www.etsi.org/committee/its) specification transcription provides a ready to connect project for the mobility (connected and autonomous vehicles, road side units, vulnerable road users,...).
 
-[workspace]
-members = [
-    "libits-client",
-    "libits-copycat",
-    "its-client",
-]
+[package]
+name = "its-client"
+version = "2.0.0"
+authors = ["Frédéric Gardes <frederic.gardes@orange.com>", "Nicolas Buffon <nicolas.buffon@orange.com>"]
+license = "MIT"
+description = "library to connect on an ITS MQTT server"
+homepage = "https://github.com/Orange-OpenSource/its-client"
+documentation = "https://docs.rs/its-client"
+repository = "https://github.com/Orange-OpenSource/its-client.git"
+readme = "README.md"
+keywords = ["network", "its", "vehicle", "mqtt", "etsi"]
+edition = "2021"
```

Comment:
Maybe we can bump that to 2023?

Ditto below, for the `[lib]` section.

Reply:
Nope, this key tells the [Rust edition](https://doc.rust-lang.org/edition-guide/index.html) to use  
The latest is 2021

(TLDR; it is the Rust way to introduce breaking changes)